app/commands/user/reputation_token/award_for_pull_request_author.rb

```diff
@@ -0,0 +1,49 @@
+class User
+  class ReputationToken
+    class AwardForPullRequestAuthor
+      include Mandate
+
+      initialize_with :params
+
+      def call
+        return unless has_author?
+        return unless merged?
+
+        user = User.find_by(github_username: params[:author_username])
+
+        unless user
+          # TODO: decide what to do with user that cannot be found
+          Rails.logger.error "Missing author: #{params[:author_username]}"
+          return
+        end
+
+        token = User::ReputationToken::Create.(
+          user,
+          :code_contribution,
+          level: author_reputation_level,
+          repo: params[:repo],
+          pr_node_id: params[:node_id],
+          pr_number: params[:number],
```

Comment:
I've kept the `pr_` prefix within the reputation tokens, as it might not be clear from the context that they're actually pull requests data.